web/src/lib/components/assets/thumbnail/thumbnail.svelte

```diff
@@ -96,7 +96,9 @@
             {#if disabled}
               <CheckCircle size="24" class="text-zinc-800" />
             {:else if selected}
-              <CheckCircle size="24" class="text-immich-primary" />
+              <div class="bg-[#D9DCEF] dark:bg-[#232932] rounded-full">
```

Comment:
I'm not proud of this part. At firt, I thought to use `bg-immich-primary/20` and `dark:bg-immich-dark-primary/20` but since the opacity is not 100%, we can still see the picture through the cicle icon. If there's a better solution, I would be glad to know.